sound/soc/sof/pm.c

```diff
@@ -376,3 +390,19 @@ int snd_sof_suspend_late(struct device *dev)
 	return sof_suspend(dev, !RUNTIME_PM);
 }
 EXPORT_SYMBOL(snd_sof_suspend_late);
+
+int snd_sof_prepare(struct device *dev)
```

Comment:
Can we make all the pm ops snd_sof_pm_ prefix - this can be an incremental PR/patch - that way it avoids collisions with the PCM ops (that also has prepare).